src/Sarif.UnitTests/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -59,13 +60,16 @@ private void RunTest(string testDirectory, string inputFileName, OptionallyEmitt
             string optionsNameSuffix = "_" + NormalizeOptionallyEmittedDataToString(optionallyEmittedData);
 
             string expectedFileName = inputFileName + optionsNameSuffix + ".sarif";
-            string actualFileName = inputFileName + optionsNameSuffix + ".actual.sarif";
```

Comment:
>actual [](start = 74, length = 6)

We now persist all generated files to an 'Actual' directory. This allows me to now produce a single diff command-line that compares output from the entire test suite (by comparing the expected vs. actual dirs). this is helpful when introducing changes that will flow to all test files.

Reply:
Got it. The file names of the "actual" files can now be the same as the file names of the "expected" files, because they are in different directories.

---
In reply to: [207576383](https://github.com/Microsoft/sarif-sdk/pull/984#discussion_r207576383) [](ancestors = 207576383)